insights/parsers/sysconfig.py

```diff
@@ -506,3 +509,52 @@ class VirtWhoSysconfig(SysconfigOptions):
         'A TEST'
     """
     pass
+
+
+@parser(Specs.ifcfg_static_route)
+class IfCFGStaticRoute(LegacyItemAccess, Parser):
```

Comment:
By inheriting it from `SysconfigOptions`, you do not implement any further operations to this Parser.

Reply:
acknowledged, updated the code in the latest PR